### DISCUSSION TITLE
How to use BluetoothA2DPCommon in a sketch

### DISCUSSION THREAD

**Discussion by timeylies** [pschatzmann/ESP32-A2DP#396] [Q&A] [answered]
How can I use BluetoothA2DPCommon in my sketches?
I want to use it since it has useful functions, like connect_to

**Comment by pschatzmann** [comment, marked as answer]
[BluetoothA2DPCommon](https://pschatzmann.github.io/ESP32-A2DP/html/class_bluetooth_a2_d_p_common.html) is an abstract class and contains the common methods that are used by the BluetoothA2DPSink and  BluetoothA2DPSource.

So using this class stand alone does not make a lot of sense and when you use one of the mentioned subclasses you have functionality available automatically.
